### ISSUE TITLE
What is your motivation?

### ISSUE THREAD

**Issue by Sch3lp** [OpenDevelopmentMethod/discussion#11]
This question is directed directly towards @ahmadnassri 

Here's what I _think_ I've learned:

* You're malcontent with Scrum/Agile dogma
* You develop software yourself and noticed the benefits of Open Source development
* You think because Open Source development works, you can distill key ingredients and apply them more generally (i.e. in Design, Business, ... processes)

Here's what I do not understand (yet):

* Why do you want to group (or organize) these best practices you've learned into a name or category?
* In choosing the title for your blog post as `Scrum is dead...`, you either wanted people to read your blog post really desperately, or you couldn't come up with a better title. Regardless of your intent, people will now understand Open Development Method as an alternative methodology. Is this what you wanted?

So in short, I'd like to know _Why_ **you** want to promote this new Open Development Method?

Cheers!

Tim

**Comment by danrichman** [comment]
I can't speak for @ahmadnassri, but there are some innovative companies incorporating the benefits of an open source model. 

The most notable example is GitHub itself, which was formed and thrived with an open source model—it was well known for being an amazing place to work because of it. [Zach Holman's talks](https://zachholman.com/talks) covered GitHub's own asynchronous internal open source model extensively.

**Comment by Sch3lp** [comment]
The benefits of working with an open source model I can get behind, don't get me wrong. I'm all for them, because they live and breathe a Living System.

So pursuing an open source model is one thing.
Trying to group practices under one name/category is another.

**Comment by danrichman** [comment]
I'm confused. Do you think it's problematic to group practices under a name/category? If so, can you elaborate?
